test/run-unittest.sh

```diff
@@ -39,10 +39,10 @@ FRONTEND="${FRONTEND:-}"
 
 if [ -z ${FRONTEND:-} ]; then
     if [ "$DC_BIN" == "ldc2" ]; then
-        FRONTEND=$(ldc2 --version | grep -Po "based on DMD v\K(2\.\d+\.\d)")
+        FRONTEND=$(ldc2 --version | grep 'based on DMD v2.' | sed -E -n 's/^.*DMD v(2\.[0-9]+\.[0-9]).*$/\1/p')
```

Comment:
The grep in my Windows VM complained about `grep -P` only working for some locales.